.changeset/pre.json

```diff
@@ -0,0 +1,37 @@
+{
+  "mode": "pre",
+  "tag": "beta",
+  "initialVersions": {
+    "example-nextjs": "0.1.0",
+    "example-node": "0.0.1",
+    "example-react-native": "0.0.1",
+    "example-shared": "0.1.0",
+    "example-wagmi": "0.1.0",
+    "@lens-protocol/api-bindings": "0.6.0",
+    "@lens-protocol/blockchain-bindings": "0.6.0",
+    "@lens-protocol/client": "0.3.0",
```

Comment:
the client is still 0.3.0 here

Reply:
these are all initial versions... before entering the pre-release mode.

Reply:
oh, I see, sorry!